src/modules/luis.module.ts

```diff
@@ -0,0 +1,52 @@
+import { BotModule } from "../types"
+import { getLog } from "../types/models/chats"
+
+import { User, Message, Client } from "discord.js"
+import pgPromise from "pg-promise"
+import { ParsingResult } from "chrono-node/dist/results"
+
+import MarkovChain from "markovchain"
+const markov = new MarkovChain()
+let vocab: string[] = []
+
+async function initializeMarkov() {
+    try {
+        console.log("Downloading chat data")
+        const logs = await getLog(0) // TODO: support multiple chatbots than hardcoding this
```

Comment:
Just as a question for my own understanding, what do you mean by this TODO?

Reply:
I think eventually we want to have multiple markov chains for different flavors of chat bots. For instance, we could have an adam bot, or a chu bot. For now we are just sticking with Luis and hard coding the ID